group_project/assignment_4.py

```diff
@@ -1,5 +1,5 @@
 def how_many_times(a_string, a_word):
-    pass
+    return a_string.count(a_word)
```

Comment:
👍

Reply:
@santiagobasulto  it doesn't matter that he didn't add the return 0 in case the a_word not in a_string, i.e. test is false? 
Otherwise it will return None ?